spark/src/main/scala/geotrellis/spark/split/Implicits.scala

```diff
@@ -30,5 +30,5 @@ trait Implicits {
   implicit class withProjectedExtentRDDSplitMethods[
     K: Component[?, ProjectedExtent],
     V <: CellGrid[Int]: (? => SplitMethods[V])
-  ](val self: RDD[(K, V)]) extends ProjectedExtentRDDSplitMethods[K, V]
+  ](val self: RDD[(K, V)])(implicit ev0: Raster[V] => SplitMethods[Raster[V]]) extends ProjectedExtentRDDSplitMethods[K, V]
```

Comment:
Do we need both `? => SplitMethods[V]`  and `Raster[V] => SplitMethods[Raster[V]]`

Reply:
heh, `Split` API is kind of a mess

Reply:
@echeipesh We do, or else it won't be able to find that implicit:

```
[error] /home/jake/Documents/projects/geotrellis/spark/src/main/scala/geotrellis/spark/split/Implicits.scala:33:36: No implicit view available from geotrellis.raster.Raster[V] => geotrellis.raster.split.SplitMethods[geotrellis.raster.Raster[V]].
[error]   ](val self: RDD[(K, V)]) extends ProjectedExtentRDDSplitMethods[K, V]
[error]                                    ^
[error] one error found
```

Yeah, the `split` API could definitely use some refactoring.